ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/main.yml

```diff
@@ -3,15 +3,6 @@
 # Main #
 ########
 
-###########################
-# Overide amd64 to x86_64 #
-###########################
-- name: If ansible_architecture is amd64 set it to x86_64
```

Comment:
Why is this task being removed? Our playbooks are set to detect the `x86_64` architecture, not `amd64`.

Reply:
Much like the `hostname` and `hosts_file` tasks that ive removed from the Common role, this same task appears in the `adopt_etc` role

Reply:
Ah, alright :+1: